apps/designer/app/designer/features/breakpoints/breakpoints.tsx

```diff
@@ -26,6 +25,7 @@ import {
 } from "./use-subscribe-shortcuts";
 import { ConfirmationDialog } from "./confirmation-dialog";
 import { useBreakpoints } from "~/shared/nano-states";
+import { sort } from "@webstudio-is/project";
```

Comment:
this is kinda too high level name to be exported there like this, because the name is too generic, we would probably need something like `breakpoint.sort()` or `breakpointUtils.sort()`

Reply:
I moved all utils under `utils` export, so `utils.breakpoints.sort()` for example.